src/main/java/com/volunteer/enterprise/VolunteerAppController.java

```diff
@@ -116,21 +102,25 @@ public Volunteer createVolunteer(@RequestBody Volunteer volunteer){
     }
 
     /**
-     * Add comments here
-     *
-     *
+     * DeleteMapping for /volunteer/{id} endpoint
+     *Delete volunteer with provided ID
      *
+     * @param id a unique identifier for volunteer to delete
      *
+     *  @return one of the following status codes:
      * @param id
      * @return
      */
     @DeleteMapping("/volunteer/{id/}")
     public ResponseEntity deleteVolunteer(@PathVariable("id") String id){
+        log.debug("Entering delete specimen endpoint");
         try{
-
+            log.info("Volunteer with ID" + id + "was deleted");
```

Comment:
info might be code after you deleted something so you can go back and see when it was deleted.